scripts/devcontainer/_vscode-post-start-command

```diff
@@ -2,6 +2,11 @@
 
 set -euo pipefail
 
+
+echo "Copying tree-sitter.so to backend/src/LibTreeSitter"
```

Comment:
I think this is a dependency for building LibTreeSitter in dotnet? I would imagine it would be copied as part of the msbuild fsproj file for that directory?

Reply:
Definitely this isn't the right place though.

A second place option might be the compile script.

Reply:
> I think this is a dependency for building LibTreeSitter in dotnet? 
this is a dependency for _running_ LibTreeSitter - it's the shared library that we P/Invoke with.

`tree-sitter.so` is only built once, as part of the container being set up, and then copied to that .NET project to be invoked.
Everything in `compile` seems to be done based on some action occurring, so it doesn't quite seem it'll fit.

Additionally, because `tree-sitter` is built outside of the `app` directory, I believe `compile` doesn't have access to the file for copying - I tried in a few other scripts as well, and `_vscode-post-start-command` seemed to be the only one that (1) had access and (2) only runs once.

Reply:
The copying shouldn't only run once though, it should for example be run again if you delete the dotnet Build directory and build again.

Doing it in a .fsproj file seesm most apppropriate.